Altis_Life.Altis/stringtable.xml

```diff
@@ -11245,6 +11245,7 @@
         </Key>
         <Key ID="STR_AdminMessage_Title">
             <Original>Admin Message</Original>
+            <German>Admin Nachricht</German>
```

Comment:
i don't agree. `Admin-Nachricht` refered to duden.de is wrong, as I tried to explain at the Gangkleidung decussion. `Adminnachricht` and `Admin Nachricht` are both not wrong but also not totally right. In my opinion `Adminnachricht` looks better. - German is the one language i really hate, altough I'm a native speaker.